core/src/test/java/io/questdb/griffin/engine/table/AsyncFilteredRecordCursorFactoryTest.java

```diff
@@ -663,4 +664,53 @@ public boolean getCloneSymbolTables() {
             return sqlExecutionContext.getCloneSymbolTables();
         }
     }
+
+    @Test
+    public void testDeferredSymbolInFilter2() throws Exception {
```

Comment:
is this a left-over or is it included intentionally?

Reply:
This intentional. This test is mentioned in the issue the PR is closing and it was reproducing a bug (bug in tests) that this PR is fixing. When I copied test from the issue, test name was already taken. 2 is not very creative - but Andrey wrote them, so perhaps he can suggest a new name